feat: 1.4 new stable API, deprecations of old API (#1769)
JetBrains/kotlin-web-site: pages/docs/reference/list-operations.md

```
</div>

> For removing the first and the last element, use functions [`removeFirst()`](/api/latest/jvm/stdlib/kotlin.collections/-mutable-list/remove-first.html)

Reviewer: I'd say it less imperatively:
```suggestion
For removing the first and the last element, there are handy shortcuts [`removeFirst()`](/api/latest/jvm/stdlib/kotlin.collections/-mutable-list/remove-first.html)
```
